README.md

```diff
@@ -942,6 +942,10 @@ Set the following config options to override the help link URLs on the Primary A
 helpLinks: {
   help: 'https://acme.com/help',
   forgotPassword: 'https://acme.com/forgot-password',
+  factorPageCustomLink: {
+    text: 'Need help with MFA?',
+    href: 'https://acme.com/mfa-help',
+  },
   unlock: 'https://acme.com/unlock-account',
   custom: [
```

Comment:
in the same topic as my comment above, this should probably have been named "signIn" or so? Not anything to change in this PR, just pointing it out to see if it makes sense to you and others, and we can have a discussion if needed later

Reply:
You mean replace "custom" -> "signIn", right?

Reply:
@denysoblohin-okta 
Just to be clear, this is unrelated to the change you're working and can be deal with separately.
Also it's breaking change (interface changed for external devs) hence probably meant for next major release.